SamplesDashboard/SamplesDashboard/ClientApp/src/types/samples.ts

```diff
@@ -10,9 +10,12 @@ export interface IRepositoryItem {
     issues: any;
     stargazers: any;
     forks: any;
-    url: string;
+    url: any;
```

Comment:
>url [](start = 4, length = 3)

it's now unclear which url this is.  name should probably be more specific.

Reply:
It represents the link to each repository. I'll make it more specific